docs/content/latest/explore/ysql-language-features/partitions.md

```diff
@@ -65,17 +65,26 @@ You create the actual partitions as follows:
 ```sql
 CREATE TABLE employees_2019_02 PARTITION OF employees
   FOR VALUES FROM ('2019-02-01') TO ('2019-03-01');
+```
 
+```sql
 CREATE TABLE employees_2019_03 PARTITION OF employees
   FOR VALUES FROM ('2019-03-01') TO ('2019-04-01');
+```
 
 ...
+
+```sql
 CREATE TABLE employees_2020_11 PARTITION OF employees
   FOR VALUES FROM ('2020-11-01') TO ('2020-12-01');
+```
 
+```sql
 CREATE TABLE employees_2020_12 PARTITION OF employees
   FOR VALUES FROM ('2020-12-01') TO ('2021-01-01');
-  
+```
+
+```sql
 CREATE TABLE employees_2021_01 PARTITION OF employees
   FOR VALUES FROM ('2021-01-01') TO ('2021-02-01');
 ```
```

Comment:
Right after this is a good place for a note:

````suggestion
```
{{< note title="Partition ranges" >}}

Partitioning ranges are _inclusive_ at the lower (`FROM`) bound, and _exclusive_ at the upper (`TO`) bound.

Each month range in the preceding examples includes the start of the month, but doesn't include the start of the following month. For example, the range `FROM ('2021-01-01') TO ('2021-02-01')` is a 31-day range from January 1, 2021, through January 31, 2021, but doesn't include February 1, 2021.

{{< /note >}}
````

Reply:
Added this, thanks for suggesting